bitrepository-webclient/src/main/webapp/audit-trail-service.html

```diff
@@ -20,319 +20,384 @@
   #L%
   -->
 <!DOCTYPE html>
```

Comment:
Of course changes in this file ended up being a mess because of weird indentation... If you CTRL+F for 'preservation' I think the actual changes should be clearer.

On another note: ideally I would want the JavaScript moved to a separate file to not clutter the html, but as this issue is apparent in all of the html files I felt the changes might as well be done in bulk, which would only serve to clutter the PR further, so I decided against doing it at this time.